src/media/js/main.js

```diff
@@ -52,7 +45,7 @@ function() {
     z.page.on('navigate divert', function(e, url) {
         if (url == urls.reverse('login') ||
             url == urls.reverse('unauthorized') ||
-            url == urls.reverse('fxa_authorize')) {
+            url.indexOf(urls.reverse('core/fxa_authorize')) !== -1) {
```

Comment:
I made a `permissions` module to handle this in transonic. Not sure if you want to copy that over here for now. It might be safer to do something like:

```js
var noAuthUrls = [
    urls.reverse('login'),
    urls.reverse('unauthorized'),
    urls.reverse('core/fxa_authorize'),
];
var baseUrl = utils.baseurl(url);
if (noAuthUrls.indexOf(baseUrl) !== -1) {
    return;
}
```

It seems pretty unlikely but `'/fxa-authorize/this-requires-auth'.indexOf('/fxa-authorize')` is `0`.